<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPlanesWindow</class>
 <widget class="QMainWindow" name="MenuPlanesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menú Planes</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_titulo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(63, 81, 181);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;MENU DE PLANES&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Regresar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(46, 26, 162) ;
border-style: outset;
background-color:rgb(255, 255, 255) ;
border-color:rgb(46, 26, 162);
border-width: 2px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>REGRESAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ConsultarPlan">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(46, 26, 162) ;
border-style: outset;
background-color:rgb(255, 255, 255) ;
border-color:rgb(46, 26, 162);
border-width: 2px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>CONSULTAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CrearPlan">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(46, 26, 162) ;
border-style: outset;
background-color:rgb(255, 255, 255) ;
border-color:rgb(46, 26, 162);
border-width: 2px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>NUEVO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_all">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(46, 26, 162) ;
border-style: outset;
background-color:rgb(255, 255, 255) ;
border-color:rgb(46, 26, 162);
border-width: 2px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>VER TODO</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMen_Planes">
    <property name="title">
     <string>Menú Planes</string>
    </property>
   </widget>
   <addaction name="menuMen_Planes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
